observation_portal/requestgroups/serializers.py

```diff
@@ -358,6 +401,10 @@ def validate(self, data):
                     raise serializers.ValidationError(_(str(cdbe)))
             readout_validation_helper = ModeValidationHelper('readout', instrument_type, modes['readout'])
             instrument_config = readout_validation_helper.validate(instrument_config)
+
+            validator = InstrumentTypeValidationHelper(instrument_type)
```

Comment:
maybe name this instrument_validator or general_instrument_validator or something

Reply:
Good catch, I missed that pattern